Sources/Bag.swift

```diff
@@ -6,13 +6,16 @@
 //  Copyright (c) 2014 GitHub. All rights reserved.
 //
 
-/// A uniquely identifying token for removing a value that was inserted into a
-/// Bag.
-public final class RemovalToken {}
-
 /// An unordered, non-unique collection of values of type `Element`.
 public struct Bag<Element> {
-	fileprivate var elements: ContiguousArray<BagElement<Element>> = []
+	/// A uniquely identifying token for removing a value that was inserted into a
+	/// Bag.
+	public typealias Token = UInt64
```

Comment:
We should make this a `struct`—not just a `typealias`. A type alias won't prevent someone from passing a random `UInt64` to `remove(using:)`. `Token`s shouldn't be publicly instantiable.